.github/workflows/_lints.yml

```diff
@@ -0,0 +1,83 @@
+name: Reusable workflow for lints
+
+on:
+  workflow_call:
+    inputs:
+      target:
+        required: true
+        type: string
+
+jobs:
+  clippy:
+    name: Clippy (MSRV)
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ${{ inputs.target }}
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install build dependencies
+        run: sudo apt install libudev-dev
+      - name: Run Clippy
+        uses: actions-rs/clippy-check@v1
+        with:
+          name: ${{ inputs.target }} / Clippy (MSRV)
+          token: ${{ secrets.GITHUB_TOKEN }}
+          args: >
+            --manifest-path ${{ inputs.target }}/Cargo.toml
+            --all-features
+            --all-targets
+            --
+            -D warnings
+
+  clippy-beta:
+    name: Clippy (beta)
+    runs-on: ubuntu-latest
+    continue-on-error: true
```

Comment:
I'm adding this like I do in other repos I maintain, because it's useful for checking whether there are Clippy lint changes coming down the pipeline that a future MSRV bump may encounter, without the noise of checking lints against nightly.

However, until #58 is closed, this step will silently fail, and the `Clippy (MSRV)` step _will_ be checking lints against nightly. So this step won't be useful until then, but it also won't get in the way, so we can add it now in preparation.

Reply:
It looks like the `auguwu/clippy-action` action will add source-code annotations when run. Does this mean we'll get duplicate annotations?

Reply:
For `actions-rs/clippy-check` this wasn't the case with `continue-on-error: true` AFAIR, but IDK about this action. At least from what I've seen in this PR itself, we weren't getting duplicates.